crates/ruff_linter/src/rules/flake8_pyi/rules/quoted_annotation_in_stub.rs

```diff
@@ -27,7 +27,7 @@ use crate::checkers::ast::Checker;
 /// ```
 ///
 /// ## References
-/// - [Static Typing with Python: Type Stubs](https://typing.readthedocs.io/en/latest/source/stubs.html)
+/// - [Typing documentation: Style guide](https://typing.readthedocs.io/en/latest/guides/writing_stubs.html#style-guide)
```

Comment:
The nice thing about the existing text here is that it made it clear that it was linking to a page specifically about stubs, rather than a more general typing style guide

Reply:
Fair, included in the title